packages/dds-components/src/components/SelectionControl/Checkbox/Checkbox.spec.tsx

```diff
@@ -65,5 +112,42 @@ describe('<Checkbox>', () => {
       expect(checkbox).toHaveAccessibleDescription(errorMessage);
       expect(checkbox).toBeInvalid();
     });
+    it('should pass disabled', async () => {
```

Comment:
Litt mer beskrivende navn på testen hadde vært digg 😄 
```suggestion
    it('should disable all checkboxes in a disabled checkbox group', async () => {
```